app/code/Magento/ConfigurableProduct/Test/Unit/Model/Product/Type/ConfigurableTest.php

```diff
@@ -394,6 +394,11 @@ public function testGetUsedProducts()
             ->willReturn('result');
 
         $this->productCollectionFactory->expects($this->any())->method('create')->willReturn($productCollection);
+        $this->salableProcessor
+            ->expects($this->once())
+            ->method('process')
+            ->with($productCollection)
```

Comment:
process() method is usually called with its second parameter as false, but test uses default method second param value, that is true.
Change `->with($productCollection)` with `->willReturnMap([
                [$productCollection, false, $productCollection]
            ]);`